File: beecrowd_exercicios_em_Cpp/beginner/area.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main(){

    double a,b,c;

    cin >> a >> b >> c;

    cout << fixed << setprecision(3);
    cout << "TRIANGULO: " << a*c/2 << "\n" << "CIRCULO: " <<  3.14159*pow(c,2) << "\n";
    cout << "TRAPEZIO: " << (a+b)*c/2 << "\n" << "QUADRADO: " << b*b << "\n" << "RETANGULO: " << a*b  << endl;


    return 0;
}

File: beecrowd_exercicios_em_python/begginner/area.py
def get_area(x, y, z):
    return {"TRIANGULO: ": x * z / 2,
            "CIRCULO: ": z * z * 3.14159,
            "TRAPEZIO: ": (x + y) * z / 2,
            "QUADRADO: ": y ** 2,
            "RETANGULO: ": x * y,
            }


def print_values(w):
    for k in w.keys():
        print("{}{:.3f}".format(k, w[k]))


a, b, c = map(float, input().split())

area = get_area(a, b, c)

print_values(area)
